test/e2e/network/dns.go

```diff
@@ -64,7 +64,12 @@ var _ = SIGDescribe("DNS", func() {
 		validateDNSResults(f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
-	It("should resolve DNS of partial qualified names for the cluster ", func() {
+	/*
+		Release: v1.14
+		Testname: DNS, PQDN
+		Description: Create a Pod. It MUST be able to resolve partial qualified DNS entries of cluster such as kubernetes.default, kubernetes.default.svc.
+	*/
+	framework.ConformanceIt("should resolve DNS of partial qualified names for the cluster ", func() {
```

Comment:
Tag as LinuxOnly and provide explanation why it doesn't work on Windows, with link to the KEP.

Reply:
Let me create another PR to tag it as [LinuxOnly]. (Keeping Conformance Promotion away from other updates to support Conformance demotion in future events)

>why it doesn't work on Windows

https://github.com/kubernetes/kubernetes/pull/74977#issuecomment-471567855

@bclau Could you please manage these Windows aspects into KEPs. (Various Windows specific discussions have been held into GitHub comments in multiple Issues/PRs but not able to track from a centralized place.)

Reply:
> @bclau Could you please manage these Windows aspects into KEPs. (Various Windows specific discussions have been held into GitHub comments in multiple Issues/PRs but not able to track from a centralized place.)

We've added Windows considerations for tests here: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#windows--linux-considerations